releases/release-1.23/release-notes/maps/pr-103724-map.yaml

```diff
@@ -1,4 +1,4 @@
 pr: 103724
 releasenote:
-  text: The image `gcr.io/kubernetes-e2e-test-images` will no longer be used in E2E
-    / CI testing, `k8s.gcr.io/e2e-test-images` will be used instead.
+  text: The image gcr.io/kubernetes-e2e-test-images will no longer be used in E2E
+    / CI testing, k8s.gcr.io/e2e-test-images will be used instead.
```

Comment:
Revert the changes here since `gcr.io/kubernetes-e2e-test-images` and  `k8s.gcr.io/e2e-test-images` better be quoted.